src/appleseed.studio/mainwindow/mainwindow.cpp

```diff
@@ -201,7 +201,7 @@ void MainWindow::new_project()
     on_project_change();
 }
 
-void MainWindow::open_project(const QString& filepath)
+void MainWindow::open_project_sync(const QString& filepath)
```

Comment:
Lets remove the _sync suffix here and in load_project_sync too.
Everything is sync by default, the suffix does not add any extra information.